system/core/language.php

```diff
@@ -1,10 +1,10 @@
 <?php
-	/*
-		Function: __
+	/**
 		Returns a string from the selected dictionary.
 
-		Parameters:
-		$msgid - identifier used as a key to fetch the corresponding string from the dictionary.
+		@param string $msgid identifier used as a key to fetch the corresponding string from the dictionary.
+		@param null|array $args arguments to be interpolated into translated string
+		@return string
```

Comment:
The comment block is not correctly formatted

```
/**
 * <--
 */
```

Reply:
Fixed.